src/main/java/com/onevizion/scmdb/SqlScriptExecutor.java

```diff
@@ -90,6 +90,37 @@ public int execute(SqlScript script) {
         }
     }
 
+    public int compileSchema() {
```

Comment:
this method is almost duplicate of `public int execute(SqlScript script)`, so I propose to add `useScriptWrapper` parameter to `execute(SqlScript script, boolean useScriptWrapper)` and use it to execute script. Then add overridden version of `execute(SqlScript script)`

Reply:
I've added the new boolean parameter `boolean needCompileSchemas` to this method which changes getting tmp wrapper script. Then added new method `executeCompileSchemas()` which allows to execute schemas compiling if necessary.